main.tf

```diff
@@ -133,10 +202,65 @@ data "aws_iam_policy_document" "resource_full_access" {
   }
 }
 
+data "aws_iam_policy_document" "resource_full_lambda_access" {
```

Comment:
instead of creating a new policy document, you can use a dynamic to extend the above document